src/components/v2v3/V2V3Project/V2V3FundingCycleSection/FundingCycleDetails/DiffedItem.tsx

```diff
@@ -0,0 +1,57 @@
+import {
```

Comment:
**suggestion** i think this component can be in the `shared` directory, because its nice and generic.